catalog/database.go

```diff
@@ -171,10 +171,8 @@ func (d *Database) createAllTable(ctx *sql.Context, name string, schema sql.Prim
 	}
 
 	// https://github.com/apecloud/myduckserver/issues/272
```

Comment:
Can this comment be removed now?

Reply:
This is a recording issue and we may rollback these changes in the future.